core/src/main/scala/org/genivi/sota/core/VehicleUpdatesResource.scala

```diff
@@ -174,8 +185,10 @@ class VehicleUpdatesResource(db : Database, resolverClient: ExternalResolverClie
   val route = {
     (pathPrefix("api" / "v1" / "vehicle_updates") & extractVin) { vin =>
       get {
-        pathEnd { logVehicleSeen(vin) { pendingPackages(vin) } } ~
-        path("queued") { pendingPackages(vin) } ~
+        namespaceExtractor { ns =>
```

Comment:
This PR uses `namespaceExtractor` which fails when the web app can't provide one (discussion in #422)

The fix put forward in #428 can also be of use here.